[2025-01-31T12:09:44.918+0700] {processor.py:186} INFO - Started process (PID=11842) to work on /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-31T12:09:44.920+0700] {processor.py:914} INFO - Processing file /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-31T12:09:44.922+0700] {logging_mixin.py:190} INFO - [2025-01-31T12:09:44.922+0700] {dagbag.py:588} INFO - Filling up the DagBag from /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-31T12:09:44.931+0700] {logging_mixin.py:190} INFO - [2025-01-31T12:09:44.930+0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-31T12:09:44.932+0700] {logging_mixin.py:190} INFO - [2025-01-31T12:09:44.932+0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-31T12:09:44.933+0700] {processor.py:927} WARNING - No viable dags retrieved from /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-31T12:09:44.973+0700] {processor.py:208} INFO - Processing /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2025-01-31T12:19:44.330+0700] {processor.py:186} INFO - Started process (PID=13388) to work on /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-31T12:19:44.331+0700] {processor.py:914} INFO - Processing file /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-01-31T12:19:44.333+0700] {logging_mixin.py:190} INFO - [2025-01-31T12:19:44.332+0700] {dagbag.py:588} INFO - Filling up the DagBag from /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-31T12:19:44.336+0700] {logging_mixin.py:190} INFO - [2025-01-31T12:19:44.335+0700] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-01-31T12:19:44.336+0700] {logging_mixin.py:190} INFO - [2025-01-31T12:19:44.336+0700] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-01-31T12:19:44.336+0700] {processor.py:927} WARNING - No viable dags retrieved from /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-01-31T12:19:44.348+0700] {processor.py:208} INFO - Processing /home/vmphat/miniconda3/envs/ds-env/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
